plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -73,7 +73,7 @@
       <launchConfigurationTabGroup
           id="com.google.cloud.tools.eclipse.appengine.localserver.AppToolsTabGroup"
           type="com.google.cloud.tools.eclipse.AppToolsLaunchConfigurationType"
```

Comment:
Possibly, but since this isn't related to tab groups and isn't user visible I'd like to leave it out of this PR.